### ISSUE TITLE
Doesnt work with some elements of Pop!_Os

### ISSUE THREAD

**Issue by Agent47UG** [EliverLara/Nordic#178]
Great theme!, but it doesnt work with the latest update of Pop_Os 21.10. The Cosmic dock and Application Menus are not supported

**Comment by AndreBClark** [comment]
I'm currently working on an update to the dock, app menus, top bar for pop 21.10 . any other areas that aren't displayed properly with 21.10?

**Comment by Agent47UG** [comment]
No, everything else works perfectly. The border of Workspace window previews is "orange" thats it. Thanks for the great theme guys <3.

**Comment by AndreBClark** [comment]
The easiest way I have found to avoid the orange highlight with workspaces was to disable the cosmic workspaces extension. for now it doesn't seem to be properly theme-able. which is proving to be pattern with the new cosmic components unfortunately,
same thing applies for the cosmic dock use the dash to dock extension and it should work as expected. in terms of the applications menu I have been unable to alter the pop-dark background and text color alterations pop performs automatically without altering pop-cosmic directly to prevent style injection. at least that's what I have discovered so far. I have suggested a fix with this pull request https://github.com/pop-os/cosmic/pull/278. I'm at an impasse unfortunately.

**Comment by AndreBClark** [comment]
unfortunately I believe this cannot be resolved without action from the pop-os team. so this may have to be tagged as 'won't fix'

**Comment by Mr-Hyde** [comment]
Hi!
Thank you for your great work. Has someone from pop-os team answered you? Is there at least a way we could manually overwrite any pop-os css file? It's kinda shame, that pop-os enables user themes, but at the same time forces its own css.

 ![image](https://user-images.githubusercontent.com/57315822/166126809-cfcd03ab-6b1f-42e8-9eb6-61ce19a01d67.png)

**Comment by AndreBClark** [comment]
> Hi! Thank you for your great work. Has someone from pop-os team answered you? Is there at least a way we could manually overwrite any pop-os css file? It's kinda shame, that pop-os enables user themes, but at the same time forces its own css.
> 
> ![image](https://user-images.githubusercontent.com/57315822/166126809-cfcd03ab-6b1f-42e8-9eb6-61ce19a01d67.png)

no response from pop yet. their repos don't seem very active. 
I suspect most of the devs are focused on the rust window manager that's on the way. 
the issue I created is here if you want to add your voice to it: https://github.com/pop-os/cosmic/issues/276

### Workaround
I found a janky work around of setting the appearance to dark and then swapping for the shell/user theme to force the shell to use pop-dark. this workaround doesn't use nord themes for the dropdowns background but is at least legible. with the new update for pop 22.04, there has been some updates to the user/shell theme so I'm considering taking another stab at implementing nordic theming for pop os soon
